### PR TITLE
Delete button

### PR DESCRIPTION
**Pull request by xp-vit**

### CODE REVIEW

File: src/main/res/layout/rando_item.xml [hunk 1]
```diff
@@ -35,4 +35,14 @@
                 app:oval="true" />
         </ViewSwitcher>
     </LinearLayout>
+    <Button
+        android:tag="delete_button"
+        android:layout_width="72dp"
+        android:layout_height="72dp"
+        android:background="@drawable/ic_trash_rando"
```
Comment (dimhold) [inline src/main/res/layout/rando_item.xml]: I guess we missed `ic_trash_randon.png` in this pull request

File: src/main/java/com/github/randoapp/adapter/RandoListAdapter.java [hunk 2]
```diff
@@ -160,6 +246,14 @@ private void recycleViewSwitcher(ViewSwitcher viewSwitcher) {
         viewSwitcher.setDisplayedChild(0);
     }
 
+    private void setAlpha(RoundedImageView view, float alpha) {
+        if (Build.VERSION.SDK_INT < 11) {
```
Comment (dimhold) [inline src/main/java/com/github/randoapp/adapter/RandoListAdapter.java]: I suggest to use positive case for more often scenarios when SDK >= 11

File: src/main/java/com/github/randoapp/adapter/RandoListAdapter.java [hunk 3]
```diff
@@ -296,13 +390,17 @@ public void onErrorResponse(VolleyError error) {
     }
 
     public static class ViewHolder {
+        public String randdoId = "";
```
Comment (dimhold) [inline src/main/java/com/github/randoapp/adapter/RandoListAdapter.java]: Typo:
```
randdoId
    ^
```

File: src/main/res/layout/rando_item.xml (hunk 1 above)
Comment (xp-vit) [inline src/main/res/layout/rando_item.xml]: Added

**Pull request merged**